src/Parser/Warlock/Demonology/Modules/Features/DEUptimeHoG.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+
+import {formatPercentage} from 'common/format';
+import StatisticBox from 'Main/StatisticBox';
+import SpellIcon from 'common/SpellIcon';
+
+const IMP_DURATION = 12;
+const MILLISECONDS = 1000;
+
+class DEUptimeHoG extends Analyzer{
```

Comment:
This module looks like a duplicate of `DEUptimeDreadstalkers`. I'd prefer if this could be optimized to eliminate the code duplication, but it might also be better to leave it as is if that might not be worth the increased code complexity. In that case I recommend leaving a comment in both classes to indicate their similarities.